python/tests/test_basic_io.py

```diff
@@ -28,14 +29,15 @@ def check_bit_flags(
 @pytest.mark.parametrize("size", [1, 10, 100, 1000, 1024, 4096, 4096 * 10])
 @pytest.mark.parametrize("nthreads", [1, 3, 4, 16])
 @pytest.mark.parametrize("tasksize", [199, 1024])
-def test_read_write(tmp_path, size, nthreads, tasksize):
+@pytest.mark.parametrize("memtype", [cupy, numpy])
```

Comment:
Wonder if we should use `xp` to match CuPy and others doing this

```suggestion
@pytest.mark.parametrize("xp", [cupy, numpy])
```

Reply:
Fixed